builder/README.md

```diff
@@ -1,5 +1,15 @@
 # builder
 
+Dependency tree
```

Comment:
this readme is a bit strange. I can see `Modules` to be a top level header, with a bullet for each module and a brief one liner description for each. But a dependency tree doesn't seem to make sense by itself. Let's make it a roadmap of what modules a user would care about for compile time and which ones for runtime.

Reply:
Updated

Reply:
pushed